ui/components/multichain/network-list-item/network-list-item.js

```diff
@@ -150,13 +157,15 @@ export const NetworkListItem = ({
         </Text>
       </Box>
       {renderButton()}
-      <NetworkListItemMenu
-        anchorElement={networkListItemMenuElement}
-        isOpen={networkOptionsMenuOpen}
-        onDeleteClick={onDeleteClick}
-        onEditClick={onEditClick}
-        onClose={() => setNetworkOptionsMenuOpen(false)}
-      />
+      {showEndAccessory ? (
```

Comment:
Nice customization! 🎉 

Would it be better if we had `endAccessory` of type  `PropTypes.node`, instead of bool `showEndAccessory`?

Reply:
I think we want to keep it boolean. Because we can only show the global menu button